Source/WebKit/Shared/TextAnimationType.serialization.in

```diff
@@ -22,16 +22,16 @@
 
 #if ENABLE(WRITING_TOOLS)
 
-header: "TextIndicatorStyle.h"
-[CustomHeader] enum class WebKit::TextIndicatorStyle : uint8_t {
+header: "TextAnimationType.h"
+[CustomHeader] enum class WebKit::TextAnimationType : uint8_t {
     Initial,
     Source,
```

Comment:
what's the difference between Initial and Source?

Reply:
Initial is the first "sweep" effect, "source" is the "sweep down" effect. These match the names in Writing Tools